bundle/android/strings.xml

```diff
@@ -157,14 +154,11 @@
     <!-- translators: accessibility text. %s: gallery caption. -->
     <string name="gutenberg_native_gallery_caption_s" tools:ignore="UnusedResources">Gallery caption. %s</string>
     <string name="gutenberg_native_give_it_a_try_by_adding_a_few_blocks_to_your_post_or_page" tools:ignore="UnusedResources">Give it a try by adding a few blocks to your post or page!</string>
-    <string name="gutenberg_native_go_back" tools:ignore="UnusedResources">Go back</string>
     <string name="gutenberg_native_gradient_type" tools:ignore="UnusedResources">Gradient Type</string>
     <string name="gutenberg_native_help_button" tools:ignore="UnusedResources">Help button</string>
     <string name="gutenberg_native_help_icon" tools:ignore="UnusedResources">Help icon</string>
     <string name="gutenberg_native_hide_keyboard" tools:ignore="UnusedResources">Hide keyboard</string>
     <string name="gutenberg_native_hide_search_heading" tools:ignore="UnusedResources">Hide search heading</string>
-    <string name="gutenberg_native_how_to_edit_your_page" tools:ignore="UnusedResources">How to edit your page</string>
-    <string name="gutenberg_native_how_to_edit_your_post" tools:ignore="UnusedResources">How to edit your post</string>
```

Comment:
I believe these string removals are erroneous, and related to the same issue @geriux encountered and manually fixed in https://github.com/wordpress-mobile/gutenberg-mobile/commit/f0038148cf79f496da0d9aad80f29306336f1454. Ideally we do not remove them, but I suppose there would be no direct harm in removing them temporarily provided we reinstate them before the next release. WDYT? 

I believe the long-term fix is removing [inline comments](https://github.com/WordPress/gutenberg/blob/641060d92e2723d82144ae1206828db7e19d3cf4/packages/editor/src/components/editor-help/index.native.js#L112) found inside of the Help section render method.

Reply:
It actually looks like the core issue causing this was a parsing failure in upstream projects. It may makes most sense not merge the string removals. WDYT? 

* https://github.com/wp-cli/i18n-command/issues/280
* https://github.com/wp-cli/i18n-command/issues/281
* https://github.com/mck89/peast/issues/42
* https://github.com/mck89/peast/issues/43

Reply:
Thanks for catching these. Does this happen with the Dependabot PRs? I was thinking about closing this PR (since I was mostly interested in running the CI tests) and rebasing the dependabot PR https://github.com/wordpress-mobile/gutenberg-mobile/pull/3864

Reply:
No, I do not believe dependabot PRs commit a new bundle, so the erroneous removal would not occur. Closing this PR and rebasing the existing dependabot PR is likely a good approach. 👍🏻